common/internal_model/src/main/scala/weco/catalogue/internal_model/work/Work.scala

```diff
@@ -233,15 +233,15 @@ object WorkState {
     canonicalId: CanonicalId,
     mergedTime: Instant,
     sourceModifiedTime: Instant,
-    availabilities: Set[Availability] = Set.empty,
-    relations: Relations = Relations.none
+    availabilities: Set[Availability] = Set.empty
   ) extends WorkState {
 
     type WorkDataState = DataState.Identified
     type TransitionArgs = Instant
 
     def id: String = canonicalId.toString
 
+    override val relations: Relations = Relations.none
```

Comment:
Do we need Relations on the core Work model at this point?

Reply:
Maybe? I feel that it could and probably should exist only in WorkState.Denormalised and WorkState.Indexed, but I'm not sure I would do this change in this PR. This is mostly to just get the router back to what it was doing before

Reply:
Sounds good, let's park it as something to address later.

(I hope we can get a TEI pipeline out today; I don't want to throw in more internal model changes that aren't blocking this work.)